pkg/sdk/procedures_validations_gen.go

```diff
@@ -29,9 +29,6 @@ func (opts *CreateForJavaProcedureOptions) validate() error {
 	if !valueSet(opts.Handler) {
 		errs = append(errs, errNotSet("CreateForJavaProcedureOptions", "Handler"))
 	}
-	if !valueSet(opts.Packages) {
```

Comment:
don't remove packages, they're required - https://docs.snowflake.com/en/sql-reference/sql/create-procedure?utm_source=snowscope&utm_medium=serp&utm_term=create+procedure#java
Same for other creates

Reply:
added validation